docs/contributing/documentation/setup-build.md

```diff
@@ -22,7 +22,7 @@ Installation of Plone 6 Documentation includes prerequisites and the repository
 
 ```{include} ../../volto/contributing/install-operating-system.md
 ```
--   {ref}`setup-build-installation-python-label` {SUPPORTED_PYTHON_VERSIONS}
+-   {ref}`setup-build-installation-python-label` {SUPPORTED_PYTHON_VERSIONS_PLONE60}
```

Comment:
I'll leave this one to you @stevepiercy, but really, you want to include Python 3.8 and 3.9 in the recommendation for building the docs?

Reply:
It's not a recommendation, but what is supported at this time with Plone 6.0. When I apply [Plone Sphinx Theme, then I will drop 3.8](https://github.com/plone/plone-sphinx-theme/blob/main/pyproject.toml#L30-L33). Or when Plone 6.1 final is released, then I'll drop 3.8, and maybe 3.9, depending on the Python requirement of plone.api and plone.restapi at that time.

Reply:
Actually, see previous comment. I think we can just go ahead with this:

```md
-   Plone 6.0 requires Python version {SUPPORTED_PYTHON_VERSIONS_PLONE60}.
-   Plone 6.1 requires Python version {SUPPORTED_PYTHON_VERSIONS_PLONE61}.
```